### PR TITLE
overhaul hebits search plugin

### PR DESCRIPTION
**Pull request by DirectorX**
### Motivation for changes:

recently hebits had transition to gazelle based site

### Config usage if relevant (new plugin or updated schema):
```yaml
  trakt-movies-search:
    template:
      - global
      - movie
    priority: 10 # run after the movie queue fill task
    discover:
      what:
        - movie_list: movies
      from:
        - hebits:
            userid: 1337
            session: "1CIecR845NxF8qmahs6sS4oGTbSP5USmHYiL8U3uaD108zL2RW7R4LIngCbqdCRG"
            category: movies

      interval: 1 hour
    tmdb_lookup: yes
    list_match:
      from:
        - movie_list: movies
```

### CODE REVIEW

File: flexget/components/sites/sites/hebits.py [hunk 1]
```diff
@@ -27,220 +17,151 @@
...
+    random = 7
 
 
 class SearchHeBits:
-    """
-    HEBits search plugin.
-    """
+    """HeBits search plugin.
+
+    ```yaml
```
Comment (paranoidi) [inline flexget/components/sites/sites/hebits.py:46]: These are only visible from commandline help. You can't use formatting and should keep it short (no complete examples with other plugins).

**Review by paranoidi — CHANGES_REQUESTED** [review]
Please fix docstring, looks good otherwise :)

**Comment by DirectorX** [comment]
ping @paranoidi

**Comment by DirectorX** [comment]
ping @paranoidi

**Comment by gazpachoking** [comment]
Thanks!

**Pull request merged**